### ISSUE TITLE
Deprecated?

### ISSUE THREAD

**Issue by raulghm** [simonsmith/gulp-suitcss#2]
Running `npm install gulp-suitcss` say:

```
npm WARN deprecated rework-suit@4.0.0: This package is deprecated - Use suitcss-preprocessor or postcss

Any help?
```

**Comment by simonsmith** [comment]
Sorry for the lack of progress here. This will be resolved in an upcoming release

**Comment by rocketspops** [comment]
@simonsmith Looks like you've made some updates, but when I try to npm install I get the same message that @raulghm did on Apr 4. Any ideas?

**Comment by simonsmith** [comment]
Hey. I haven't made a new release just yet as we're about to release 3.0.0 of the preprocessor which this plugin will work with. It'll be very soon :)

**Comment by rocketspops** [comment]
@simonsmith Great, I look forward to it. Thanks for the update. For now, I'm installing this plugin directly from your repo and it's working just fine.

**Comment by simonsmith** [comment]
Released [2.0.0](https://github.com/simonsmith/gulp-suitcss/releases/tag/2.0.0)
